src/components/Pagination/Pagination.test.jsx

```diff
@@ -120,12 +119,12 @@ describe('<Pagination />', () => {
     });
     it('is disabled when page is first page', () => {
       expect(previousLink).toHaveLength(1);
-      expect(previousLink.text()).toEqual('previous button disabled');
+      expect(previousLink.text()).toEqual('previousbutton is disabled');
```

Comment:
Why no more spaces?

Reply:
This string previously had an odd space in it: https://github.com/edx/studio-frontend/pull/84/files/#diff-19113b55aba41492c4d2adb0959b1478L23

The full "sentence" isn't really a sentence at all, it's the `previous` label with some follow-up sr-only text, so the text() return value is smashed together like this. 
<img width="434" alt="screen shot 2018-02-05 at 12 35 36 pm" src="https://user-images.githubusercontent.com/7373924/35819643-4c988e18-0a71-11e8-9c86-d62744a39e16.png">